Python/ogusa/macro_output.py

```diff
@@ -94,9 +94,9 @@ def dump_diff_output(baseline_dir, policy_dir):
     pct_changes[:,10] = ((policy_macros[:,:10].sum(axis=1)-baseline_macros[:,:10].sum(axis=1))/policy_macros[:,:10].sum(axis=1))
     
     ## Load SS results
-    ss_policy_path = os.path.join(policy_dir, "SS", "ss_vars.pkl")
+    ss_policy_path = os.path.join(policy_dir, "SS", "SS_vars.pkl")
```

Comment:
@jdebacker @rickecon can you confirm this change is appropriate? I changed it because the code in `execute.py` here:

https://github.com/open-source-economics/OG-USA/blob/master/Python/execute.py#L241

now looks for a filed named `SS_vars.pkl` instead of the lower-case `ss_vars.pkl`. This branch works on my test node but just want to make sure. thanks!